File: oj.uz/IOI17_train/java/train.java
public class train {

	public int[] who_wins(int[] a, int[] r, int[] u, int[] v) {
		int[] res = new int[a.length];
		for(int i = 0; i < a.length; i++)
			res[i] = i % 2;
		return res;
	}
}

File: oj.uz/IOI17_train/pas/train.pas
unit train;

interface

type
	TIntArray = array of longint;

function who_wins(a, r, u, v: TIntArray): TIntArray;

implementation

function who_wins(a, r, u, v: TIntArray): TIntArray;
var
	res: TIntArray;
	i: longint;
begin
	setLength(res, length(a));
	for i := 0 to length(res) - 1 do
		res[i] := i mod 2;
	who_wins := res;
end;

end.
